docs/en/02_Developer_Guides/01_Templates/02_Common_Variables.md

```diff
@@ -368,9 +368,9 @@ of the `silverstripe/cms` module.
 ```
 
 <div class="info" markdown="1">
-To customise the markup that the `$Breadcrumbs` generates, copy `templates/BreadcrumbsTemplate.ss`
- from the `silverstripe/cms` module to 
-`app/templates/BreadcrumbsTemplate.ss`, modify the newly copied template and flush your SilverStripe cache.
+To customise the markup that `$Breadcrumbs` generates, copy `templates/BreadcrumbsTemplate.ss`
+ from the `silverstripe/cms` module to your theme (e.g.: `themes/simple/templates/BreadcrumbsTemplate.ss`).
```

Comment:
My gripe with this is that the simple theme is a composer dependency of a default install, maybe "your-theme" would be better?

Reply:
```suggestion
 from the `silverstripe/cms` module to your theme (e.g.: `themes/you-theme/templates/BreadcrumbsTemplate.ss`).
```